pkg/cmd/dump.go

```diff
@@ -173,13 +174,13 @@ func dumpNamespace(ctx context.Context, c client.Client, ns string, out *os.File
 	return nil
 }
 
-func dumpConditions(prefix string, conditions []v1.PodCondition, out *os.File) {
+func dumpConditions(prefix string, conditions []v1.PodCondition, out io.Writer) {
```

Comment:
Any reason to change this? I don't see any of the method caller changed.

Reply:
[Here](https://github.com/apache/camel-k/blob/2be1a0a450413be14939deb8a623ddd5cc01aef8/pkg/cmd/dump.go#L160) io.Writer will be passed to this function because [this](https://github.com/apache/camel-k/blob/2be1a0a450413be14939deb8a623ddd5cc01aef8/pkg/cmd/dump.go#L76) function definition was also changed due to the fact that dump output could be redirected to the [os.File](https://github.com/apache/camel-k/blob/2be1a0a450413be14939deb8a623ddd5cc01aef8/pkg/cmd/dump.go#L62-L72) also.

Reply:
Ah okey, I've missed it, thanks!